app/javascript/article-form/components/Help/BasicEditor.jsx

```diff
@@ -31,5 +31,5 @@ export const BasicEditor = ({ openModal }) => (
 );
 
 BasicEditor.propTypes = {
-  toggleModal: PropTypes.func.isRequired,
+  openModal: PropTypes.func.isRequired,
```

Comment:
Cypress tests were randomly failing until I updated this PropType name from `toggleModal` (which is not used in this file) to `openModal` 🤷🏾‍♀️

Reply:
I don't think prop types are validated at all in cypress or jest specs so I'm not sure why things were failing 🙃 doesn't hurt to correct it though!